pkg/apis/cloudcredential/v1/types_credentialsrequest.go

```diff
@@ -60,6 +60,12 @@ type CredentialsRequestSpec struct {
 	// credentials flow.
 	// +optional
 	ServiceAccountNames []string `json:"serviceAccountNames,omitempty"`
+	// CloudTokenString (for instance for AWS a Role ARN)
+	// +optional
+	CloudTokenString string `json:"cloudTokenString,omitempty"`
+	// CloudTokenPath (JWT token)
+	// +optional
+	CloudTokenPath string `json:"cloudTokenPath,omitempty"`
```

Comment:
@bentito do we want the path configurable since today its not in CCO

Reply:
@jharrington22 I can't quite tell from how you are asking:

Do you think we should make the path configurable? If so, is it enough that it's configurable in the CredentialsRequest? Or did you mean it should just be a well-known-location?